src/pybamm/models/submodels/interface/sei/sei_growth.py

```diff
@@ -135,6 +137,26 @@ def get_coupled_variables(self, variables):
             # Scott Marquis thesis (eq. 5.94)
             eta_inner = delta_phi - phase_param.U_inner
             j_sei = (eta_inner < 0) * phase_param.kappa_inner * eta_inner / L_sei_inner
+
+        elif SEI_option == "Kolzenberg2020":
+            # Equation 19 in
+            # von Kolzenberg L, Latz A, Horstmann B.
+            # Solid electrolyte interphase during battery cycling:
+            # Theory of growth regimes. ChemSusChem. 2020 Aug 7;13(15):3901-10.
+            eta_bar = F_RT * (delta_phi)
+            L_diff = (
+                phase_param.D_li * phase_param.c_li_0 * param.F / phase_param.j0_sei
+            ) * pybamm.exp(-(1 - alpha_SEI) * eta_bar)
+            L_tun = phase_param.L_tunneling
+            L_app = (L_sei - L_tun) * ((L_sei - L_tun) > 0)
+            L_mig = (
+                2 / F_RT * phase_param.kappa_Li_ion / (pybamm.AbsoluteValue(j) + 1e-38)
```

Comment:
Maybe we could use `pybamm.max()` instead, like we do in other places. This way you can also set a larger tolerance if you want as errors won't accumulate.